artemis-commons/src/main/java/org/apache/activemq/artemis/utils/OrderedExecutorFactory.java

```diff
@@ -104,7 +103,7 @@ public void run() {
                         // This could happen during shutdowns. Nothing to be concerned about here
                         logger.debug("Interrupted Thread", e);
                      } catch (Throwable t) {
-                        ActiveMQClientLogger.LOGGER.caughtunexpectedThrowable(t);
+                        logger.warn(t.getMessage(), t);
```

Comment:
nope... it's just a generic exception, it can just call logger.warn directly
if were doing anything with it, such as adding our messages.. then it would be through the internationalized Logger.